test/cuda/Makefile

```diff
@@ -0,0 +1,16 @@
+include $(HCPP_ROOT)/include/hcpp.mak
+
+TARGETS=allocate copy kernel
+
+FLAGS=-g -std=c++11
+
+all: $(TARGETS)
+
+kernel: kernel.cu
+	nvcc -arch=sm_30 ${FLAGS} $(PROJECT_CFLAGS) $(PROJECT_LDFLAGS) -o $@ $^ $(PROJECT_LDLIBS)
```

Comment:
It could be, but it shouldn't. Forcing -arch=sm_30 means that there are a lot of GPUs this wouldn't even compile for, but which we could support.